### PR TITLE
Flamethrower Turret with image, attributes, and unlock missions

### PR DESCRIPTION
**Pull request by ItsNickBarry**
In response to #947 by @DearVarun, I've added an image for a flamethrower turret, made an attempt to balance it, and written the missions to unlock it.

The missions are currently not working, though they mirror the structure of the "FW Flamethrower" missions almost exactly.  The problem is with the mission "FW Flamethrower Turret 2".  Line 1151 specifies that an NPC should be disabled, but not killed, to complete the mission.  Killing it causes the mission to fail, as expected, but disabling it does not allow the mission to be completed.  Can anyone see that I've made a mistake?  I can find nothing, and am beginning to think that it's a problem with the "disable" tag.

@endless-sky What do you think of the image?  If it doesn't meet your standards, I can modify it, or we can close this without necessarily attempting to solve the problem mentioned above.

### CODE REVIEW

**Comment by LocalGod79** [comment]
the engineer at Kraz Cybernetics is 'Barmy Edward'.  Dr. Cardona refers to him by "Edward" (sans-Mr.) as his first name.

perhaps make it 'npc save board' in the meantime?

**Comment by DingusShingleton** [comment]
Did you intend for the flamethrower turret to generate less firing heat than the flamethrower? The gun has 2 firing heat (120/sec) and the turret as listed now has 1.5 (90/sec). Should be more, not less, no?

**Comment by toilethinges** [comment]
The image makes it appear as if it came with fuel - perhaps add 10 or so fuel capacity just for fun?

**Comment by ItsNickBarry** [comment]
@LocalGod79 I was trying to make Dr. Cardona seem curt and to-the-point by having him refer to Barmy by last name only.  It didn't work here because "Edward" is also a first name.

@DingusShingleton DearVarun had set the reload rate to .5, so the firing heat per second would have been doubled.  However, a reload rate of less than 1 didn't seem to be accounted for in the "heat damage per second" calculation, so I instead gave the turret its own submunition type with double the damage.  In-game flamethrower and turret firing heats are 120 and 180, respectively.

@toilethinges It has a fuel capacity of 100, like a standard fuel pod.

**Comment by toilethinges** [comment]
@ItsNickBarry - Apologies, I was looking at the wrong section of attributes. Looks good to me!

**Comment by endless-sky** [comment]
"Edward" *is* his first name. "Barmy" is an adjective.

> Foolish; eccentric; mildly crazy; cracked. First form chiefly British 1600+, second 1800s+; from barm, ''froth on fermenting beer,'' hence ''flighty, ditsy''; fell in with balmy, said to be from St Bartelemy, the patron of mad folk, perhaps because the words are homophones in British English.

File: data/free worlds start.txt [hunk 1]
```diff
@@ -1103,6 +1103,83 @@ mission "FW Flamethrower 2"
...
+			label good
+			`	"Glad to hear it."`
+			`	He glances at the figures displayed on his monitor.`
+			`	"Yes, the data we've collected remotely seem to meet our expectations."`
+				goto end
+			label bad
+			`	"I see. Surely that speaks more to the effectiveness of your ship's cooling system than to any deficiency in the prototype."`
+			choice
+				`	"Indeed."`
+				`	"Don't call me Shirley."`
```
Comment (Gkramme) [inline data/free worlds start.txt]: Where does the name Shirley come from? It doesn't seem to fit with the way the game presents the main character (you).

**Comment by ItsNickBarry** [comment]
@Gkramme It's a [joke](https://www.youtube.com/watch?v=KM2K7sV-K74).

**Comment by ItsNickBarry** [comment]
@endless-sky I've tested this against your fix, and it's working.

**Comment by ItsNickBarry** [comment]
@endless-sky Anything you want me to do with this?

**Comment by endless-sky** [comment]
I'm just not sure this belongs in the base game. The basic flamethrower is already, in some ways, way overpowered and difficult to balance with other weapons, and adding a turret version just makes that worse.

**Comment by ItsNickBarry** [comment]
I have released this as a [plugin](https://github.com/ItsNickBarry/endless-sky-flamethrower-turret).

**Pull request closed without merging**